cpp/src/parquet/metadata.h

```diff
@@ -306,9 +306,15 @@ class PARQUET_EXPORT FileMetaData {
   int num_schema_elements() const;
 
   /// \brief The total number of rows.
+  ///
+  /// If the FileMetaData is generate from a `SubSet()`, the number of rows
+  /// will be the sum of the number of rows of remaining RowGroup.
```

Comment:
This is a bit confusing. How about:
```suggestion
  /// If the FileMetaData was obtained by calling `SubSet()`, this is the total
  /// number of rows in the selected row groups.
```